code/src/components/Album.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { AlbumArtwork } from './AlbumArtwork'
+import { ArtistName } from './ArtistName'
+import { AlbumName } from './AlbumName'
+
+export const Album = (props) => {
+  return (
+    <>
+      <AlbumArtwork
+        key={props.id}
```

Comment:
Same as in previous comment - there is no need to add `key` attribute over here.
Small tip for you - you can always check it in Console tab inside dev tools. Try to remove `key` one by one from every place you added it to and check if new warning will be displayed in Console tab. If yes, then you will be 100% sure you need this `key`. If not, you are good to go :)